web/themes/contrib/civictheme/assets/sass/theme.scss

```diff
@@ -7,3 +7,4 @@
 // These overrides will not be visible in the Storybook.
 @import 'page/page';
 @import 'header/header';
+@import 'ckeditor/ckeditor';
```

Comment:
The issue is specific to Drupal’s creditor classes, and therefore, it was more appropriate to handle it as a Drupal-specific override. Hence not added via UIKit.

Reply:
I think it makes sense to have this code specific to Drupal.